docker/docker-compose.ci.yml

```diff
@@ -57,10 +57,13 @@ services:
     image: unlock-app
     build:
       context: ../
-      dockerfile: ./docker/unlock-app.dockerfile
+      args: 
+        BUILD_DIR: unlock-app
       cache_from:
         - unlock-app
         - unlockprotocol/unlock-app:master
+    depends_on:
+      - ganache-integration
```

Comment:
Needed?

Reply:
yes i think so, we better make sure ganache is running before we start the tests -- even tho jest is likely slower to start